app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    total_rating = [player.get_rating() for player in team]
```

Comment:
The list comprehension `[player.get_rating() for player in team]` is unnecessary here. According to the checklist, you should prevent creating a redundant list by using a generator expression instead. You can change this line to `total_rating = (player.get_rating() for player in team)` to improve efficiency.